<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vnc</class>
 <widget class="QDialog" name="vnc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>544</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Configurar VNC</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recoverdrake.qrc">
    <normaloff>:/Imagenes/vnc.png</normaloff>:/Imagenes/vnc.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="3" column="0" colspan="6">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Password para acceder con conexion remota</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_11">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>7</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Introducir password:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_11">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="recoverdrake.qrc">:/Imagenes/good.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>7</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Confirmar password:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_12">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="recoverdrake.qrc">:/Imagenes/good.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="checkBox_6">
          <property name="font">
           <font>
            <pointsize>7</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Visualizar password</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>7</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Datos de e-mail para envio de ip</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_16">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">font: 6pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 255);
font: bold;</string>
          </property>
          <property name="text">
           <string>*Tiene que ser una cuenta de gmail</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_7">
          <property name="font">
           <font>
            <pointsize>7</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Crear cuenta</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="recoverdrake.qrc">:/Imagenes/good.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_6">
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Configurar</string>
            </property>
            <property name="icon">
             <iconset resource="recoverdrake.qrc">
              <normaloff>:/Imagenes/nucleo.png</normaloff>:/Imagenes/nucleo.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="checkBox_7">
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Visualizar clave</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_14">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>e-mail</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="whatsThis">
             <string>Si activas esta opcion tendras que abrir los puertos de tu router desde 5900 al 5910 minimo</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>@gmail.com</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>smtp</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_5">
            <property name="text">
             <string>smtp.gmail.com</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Clave</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_6">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_15">
          <item row="0" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>e-mail receptor</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_8"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="4">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Tipo de IP publica para servidor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="radioButton">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>7</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>IP fija (Estatica) [Por defecto]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="whatsThis">
         <string>Si activas esta opcion tendras que abrir los puertos de tu router desde 5900 al 5910 minimo</string>
        </property>
        <property name="text">
         <string>IP variable (Dinamica)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 6pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 255);
font: bold;</string>
        </property>
        <property name="text">
         <string>* Si no sabes que contestar te aconsejo que selecciones IP variable (Dinamica)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>7</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>IP publica</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_7">
          <property name="styleSheet">
           <string notr="true">font: 75 10pt &quot;Sans Serif&quot;;
font: bold;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QPushButton" name="pushButton_2">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>33</height>
      </size>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
     <property name="icon">
      <iconset resource="recoverdrake.qrc">
       <normaloff>:/Imagenes/puerta.png</normaloff>:/Imagenes/puerta.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="maximumSize">
      <size>
       <width>89</width>
       <height>33</height>
      </size>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
     <property name="icon">
      <iconset resource="recoverdrake.qrc">
       <normaloff>:/Imagenes/guardar.png</normaloff>:/Imagenes/guardar.png</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>297</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">font: 6pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 255);
font: bold;</string>
     </property>
     <property name="text">
      <string>* Si se marca &quot;Limitar acceso a lista de ip's&quot; no podras acceder con ip remota dinamica.</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>Por Defecto</string>
     </property>
     <property name="icon">
      <iconset resource="recoverdrake.qrc">
       <normaloff>:/Imagenes/sustituir.png</normaloff>:/Imagenes/sustituir.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Sans Serif&quot;;
font: bold;</string>
     </property>
     <property name="text">
      <string>Datos de configuracion del servidor VNC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Lista de ip's aceptadas</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="3">
         <widget class="QTableView" name="tableView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_3">
          <property name="maximumSize">
           <size>
            <width>84</width>
            <height>26</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>7</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Insertar</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/thumbs_up.png</normaloff>:/Imagenes/thumbs_up.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_4">
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>26</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>7</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Eliminar</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/bad.png</normaloff>:/Imagenes/bad.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>7</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Opciones de configuracion</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Servidor en segundo plano</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Iniciar menu grafico</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>Siempre escuchando peticiones</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Solo se puede visualizar</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>Limitar acceso a lista de ip's</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recoverdrake.qrc"/>
 </resources>
 <connections/>
</ui>
